lib/class-wp-rest-menus-controller.php

```diff
@@ -485,6 +590,12 @@ public function get_item_schema() {
 			'context'     => array( 'view', 'edit' ),
 		);
 
+		$schema['properties']['auto_add'] = array(
+			'description' => __( 'Whether to automatically add top level pages to this menu.', 'gutenberg' ),
+			'context'     => array( 'view', 'edit' ),
```

Comment:
@TimothyBJacobs Thoughts on context here?

Reply:
That works for me. It follows the menu locations.